robot/Cumulus/tests/browser/gift_entry/reorder_template.robot

```diff
@@ -0,0 +1,109 @@
+  ...                                   gift form in a batch created from that template.
+  [tags]                                unstable                    feature:GE          W-039563
+  ${TEMPLATE} =                         Generate Random String
+  Go to Page                            Landing                     GE_Gift_Entry
+  Click Link                            Templates
+  Click Gift Entry Button               Create Template
+  Current Page Should Be                Template                    GE_Gift_Entry
+  Enter Value in Field
+  ...                                   Template Name=${TEMPLATE}
+  ...                                   Description=This is created by automation script 
+  Click Gift Entry Button               Next: Form Fields
+  Click Gift Entry Button               button Settings Gift Entry Form
+  Enter Value in Field                  
+  ...                                   Section Name=FormSection1
+  Click Button                          Save
+  Wait Until Element Is Not Visible     Save
+  Click Button with Title                          Add Section
+  Perform Action On Object Field        select                     AccountSoftCredits  Role
+  Perform Action On Object Field        select                     CustomObject1  CustomObject1Imported
+  Click Gift Entry Button               button Settings New Section
+  Enter Value in Field                  
+  ...                                   Section Name=FormSection2
+  Click Button                          Save
+  Wait Until Element Is Not Visible     Save
+  # Scroll Page To Location               0  900
+  Click Gift Entry Button               button Up FormSection2
+  Perform Action On Object Field        unselect                   Payment       Check/Reference Number
+  #Gets names of template builder sections and fields and stores them in a list
+  Get Template Builder Field Names
+  Get Template Builder Section Names
+  Click Gift Entry Button               Save & Close
+  Current Page Should Be       Landing         GE_Gift_Entry
+  Click Link                   Templates
+  Wait Until Page Contains     ${TEMPLATE}
+  Store Template Record Id     ${TEMPLATE}
+  Create Gift Entry Batch               ${TEMPLATE}  ${TEMPLATE} Batch
+  Current Page Should Be                Form                        Gift Entry   title=FormSection1 
+  Page Should Contain                   AccountSoftCredits: Role        
+  Page Should Not Contain               Check/Reference Number
+  #Gets names of form sections and fields, stores them in a list, and compares the order to the template builder page's list
+  Get Gift Entry Form Field Names
+  Get Gift Entry Form Section Names
+  Lists Should Be Equal                 ${BUILDER_FIELD_LABELS}  ${FORM_FIELD_LABELS}
+  Lists Should Be Equal                 ${BUILDER_S_TITLES}  ${FORM_S_TITLES}                       
+  ${BATCH_ID} =                         Save Current Record ID For Deletion      ${ns}DataImportBatch__c
```

Comment:
batch_id should be lower case as it is local variable

Reply:
Will be fixed in next push.